game/views.py

```
@@ -808,6 +1077,24 @@ def _clear_registration_session(request):
     for key in REGISTRATION_SESSION_KEYS:
         request.session.pop(key, None)
 
+@require_POST
+def cleanup_stale_games(request):
```

Ruff (0.15.20)

[warning] Unused function argument: `request`

(ARG001)